compiler/rustc_codegen_llvm/src/coverageinfo/mapgen.rs

```diff
@@ -174,8 +174,14 @@ impl GlobalFileTable {
         // Since rustc generates coverage maps with relative paths, the
         // compilation directory can be combined with the relative paths
         // to get absolute paths, if needed.
+        use rustc_session::config::RemapPathScopeComponents;
         use rustc_session::RemapFileNameExt;
-        let working_dir: &str = &tcx.sess.opts.working_dir.for_codegen(tcx.sess).to_string_lossy();
+        let working_dir: &str = &tcx
+            .sess
+            .opts
+            .working_dir
+            .for_scope(tcx.sess, RemapPathScopeComponents::DEBUGINFO)
```

Comment:
@Zalathar, I found you in triagebot.toml for coverage related questions:

Do you know where the file path generated here will end up in? Constant data? A coverage-specific file? Debuginfo?

Reply:
It ends up inside a special `__llvm_covmap` section in the final binary. LLVM's `llvm-cov` tool then reads it back out of the binary when preparing coverage reports, in order to find the corresponding source files.

Reply:
Thanks for the info, @Zalathar!

I looks like Clang has its own remapping scope for this via [-fcoverage-prefix-map](https://clang.llvm.org/docs/ClangCommandLineReference.html#cmdoption-clang-fcoverage-prefix-map). We don't have that so I think MACRO is the best fit for now.

Reply:
Also note that this path ends up in the same place as the paths created in `compiler/rustc_mir_transform/src/coverage/mod.rs`, so whatever the decision is, both should get the same treatment.

Reply:
I'm not familiar with path remapping, but it does seem very unintuitive to me that this information would be controlled by something named `macro`.

Reply:
Yes, I agree. It's a [known issue](https://github.com/rust-lang/rfcs/pull/3127/files#r882881164) that the name is not a good fit for everything that it covers. I definitely think that this should be fixed before stabilization.

Reply:
Done, switch to `MACRO`.

Btw, I like your suggestion of renaming it to `expansion`.